### PR TITLE
Move attachment email to its own mailer

### PR DESCRIPTION
**Pull request by jessieay**
* Slowly but surely moving everything out of Mailer

### CODE REVIEW

**Comment by pkarman** [comment]
@jessieay merge conflicts, alas

**Comment by jessieay** [comment]
@pkarman updated!

File: lib/mail_previews/mailer_preview.rb [hunk 1]
```diff
@@ -33,7 +29,12 @@ def proposal
     Proposal.last
   end
 
+<<<<<<< HEAD
```
Comment (pkarman) [inline lib/mail_previews/mailer_preview.rb]: looks like a merge conflict got committed @jessieay ?

Comment (jessieay) [inline lib/mail_previews/mailer_preview.rb]: oy! fixed in feac16b - thanks for the careful re-review

**Pull request merged**